src/serlo-editor-repo/plugin-rows/components/rows-editor.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react'
+
+import { RowsProps } from '..'
+import { useScopedSelector } from '../../core'
+import { getPluginTypesOnPathToRoot } from '../../store'
+import { styled } from '../../ui'
+import { useRowsConfig } from '../config'
+import { RegistryContext } from '../registry-context'
+import { RowEditor } from './row-editor'
+import { RowSeparator } from './row-separator'
+
+const ReadOnlyRow = styled.div({
+  marginBottom: '25px',
+})
+
+export function RowsEditor(props: RowsProps) {
+  const config = useRowsConfig(props.config)
+  const pluginTypesOfAncestors = useScopedSelector(
+    getPluginTypesOnPathToRoot(props.id)
+  )
+
+  function insertRowWithSuggestionsOpen(insertIndex: number) {
+    const textPluginWithSuggestions = {
+      plugin: 'text',
+      state: [{ type: 'p', children: [{ text: '/' }] }],
```

Comment:
When using a button to insert the Textplugin:
would this be possible to open the menu but not insert the "/"?

Reply:
For sure, anything is possible. But it would take a completely another approach to opening the menu then, currently the menu is open when we have the "/".